flycs_sdk/entities.py

```diff
@@ -20,6 +27,7 @@ def __init__(
         self,
         name: str,
         version: str,
+        kind : Kind = None,
```

Comment:
When the argument is optional, wrap its type with Optional to make the type system know it can be None.
```suggestion
        kind : Optional[Kind] = None,
```